tests/steps/field/test_field_merge.py

```diff
@@ -0,0 +1,55 @@
+from frictionless import Resource, transform, steps
+
+# Issues
```

Comment:
Sorry it was legacy - we don't need this comment-header (the initial intention was like `bug fixing test`)